main.go

```diff
@@ -1731,41 +1732,29 @@ func main() {
 			os.Exit(1)
 		}
 	case "monitor":
-		err := Monitor("", *port, options)
-		handleCompilerError(err)
+		if *info {
+			serialPortInfo, err := ListSerialPorts()
+			handleCompilerError(err)
+			for _, s := range serialPortInfo {
+				fmt.Printf("%s %4s %4s %s\n", s.Name, s.VID, s.PID, s.Target)
+			}
+		} else {
+			err := Monitor("", *port, options)
+			handleCompilerError(err)
+		}
 	case "targets":
-		dir := filepath.Join(goenv.Get("TINYGOROOT"), "targets")
-		entries, err := os.ReadDir(dir)
+		specs, err := getTargetSpecs()
```

Comment:
Why export `ListSerialPorts()` but not export `getTargetSpecs()`? They should both have the same level of external visibility, whatever that is, don't you think @sago35 ?

Reply:
Yes, I believe your point is valid.
I've made the corrections.
I also wanted to adjust the commit message, so I did a rebase and force-pushed.